templates/zerver/help/restore-the-last-unsent-message.md

```diff
@@ -8,5 +8,3 @@ your messaging box, you can restore your previous unsent message by
 clicking a blue link labeled **Restore draft**, which appears between
```

Comment:
"the **Restore draft** link"

Also, the first sentence is inaccurate because you can 1) only restore drafts for  UNSENT messages 2) You can accidentally discard your message through other methods, not just the X icon. Please update accordingly.

You can also crop the image since it shows some unnecessary parts.